src/Console/Command/Compile.php

```diff
@@ -585,7 +586,12 @@ private function configureCompressionAlgorithm(Configuration $config, Box $box,
             )
         );
 
-        $restoreLimit = $this->bumpOpenFileDescriptorLimit($box, $io);
+        if ($dev) {
+            // Skips the compression step in dev mode
```

Comment:
Should you provide feedback to the user that compression is being skipped because dev mode?

Reply:
Good idea 👍